src/Analysis/Codelyzer.Analysis.Common/Codelyzer.Analysis.Common.csproj

```diff
@@ -7,6 +7,7 @@
     </PropertyGroup>
 
     <ItemGroup>
+      <PackageReference Include="Microsoft.Build" Version="17.0.0" />
```

Comment:
We could but that package is currently frozen on an old version due to a new bug they introduced in version 4.* I would rather use latest solution parser from Microsoft instead of having to rely on this older package. We could change in the future and use this sub-package instead. Although this still requires adding a new package reference to this new subpackage as it does not come with the base package itself which somewhat defeats the point of trying to not add a new dependency anyway.

Reply:
It seems like the parse method is pretty efficient so it's probably the better solution anyway.